elide-core/src/main/java/com/yahoo/elide/audit/AuditLogger.java

```diff
@@ -27,7 +25,7 @@ public void log(LogMessage message) {
         messages.get().add(message);
     }
 
-    public abstract void commit(RequestScope requestScope) throws IOException;
+    public abstract void commit() throws IOException;
```

Comment:
@aklish Currently, our audit logger uses requestScope to access the transaction in order to save audit record information to the database.  Is there another way then to access request scope?